lib/chart/internal/axis.ex

```diff
@@ -29,78 +29,94 @@ defmodule Chart.Internal.Axis do
             # Internal
             line: nil
 
+  @spec new() :: Axis.t()
+  def new() do
+    %__MODULE__{
+      label: nil,
+      major_ticks: nil,
+      minor_ticks: nil,
+      major_ticks_text: nil,
+      scale: :linear,
+      thickness: 2,
+      line: nil
+    }
+  end
+
+  def put(axis, key, config_key, config) do
+    Utils.put(axis, key, config_key, config, &validate/0)
+  end
+
+  # Private
+
+  defp validate() do
+    %{
+      scale: &Validators.validate_axis_scale/1,
```

Comment:
nemel by byt tento validator spis soucasti axis?

Reply:
ee mam zato ze je, nebo spatne koukam?

Reply:
@s-m-i-t-a: nemyslim si, kdyz nahore mas alias  ...{..., Validators} a pak to pouzivas ´&Validators.validate_axis_scale/1´.

Aha, takhle to myslis. No ja bych vsechny ty validatory asi vycleni do toho modulu `Validators`

Reply:
Obecne ano, ale konkretni ne.